### ISSUE TITLE
Deleting this repository

### ISSUE THREAD

**Issue by maackle** [maackle/permablocks#1]
I realized that I have an exact copy of this repo up at https://github.com/maackle/permablocks-ui - so you can reference that from now on. Either way, this codebase is pretty much dead, and is being replaced by https://github.com/terrarium-project/diagrammer

**Comment by maackle** [comment]
Indeed, connecting inputs to outputs is not implemented yet, but it's next on the todo list.  Currently inputs are attracted to outputs and vice versa but there's no other interaction.  This whole thing is still in a _really_ rough state, but I'll be chipping away at it over the next few weeks.  Thanks for the interest, and stay in touch!

**Comment by ichaleynbin** [comment]
Will do! I might just have to pick up some coffeescript and do a bit of coding on it myself. I'm really excited about this project, it's got great implications to me. If you'd like to go over some details on what needs to be coded next, I learn best by having something to do. For example, http://ichaleynbin-todo-list.meteor.com is my first ever project in javascript, and therefore my first ever project in Meteor, but it took me only a week to complete. 

Reviewing the codebase now. If you'd like to add me to skype or something, my tag is always ichaleynbin.
